package.json

```diff
@@ -27,18 +27,18 @@
   "author": "Klaus Hartl",
   "license": "MIT",
   "devDependencies": {
-    "grunt": "1.0.0",
+    "grunt": "1.0.1",
     "grunt-compare-size": "0.4.2",
     "grunt-contrib-connect": "1.0.2",
     "grunt-contrib-jshint": "1.1.0",
     "grunt-contrib-nodeunit": "1.0.0",
-    "grunt-contrib-qunit": "1.3.0",
-    "grunt-contrib-uglify": "2.2.1",
+    "grunt-contrib-qunit": "2.0.0",
+    "grunt-contrib-uglify": "2.3.0",
     "grunt-contrib-watch": "1.0.0",
     "grunt-jscs": "3.0.1",
     "grunt-saucelabs": "9.0.0",
     "gzip-js": "0.3.2",
-    "qunitjs": "1.23.1",
+    "qunitjs": "2.3.2",
```

Comment:
We can't update qunit. See https://github.com/js-cookie/js-cookie/pull/216#issuecomment-228578080.